source/Templates/NewStep/Sashimi.NamingIsHard/source/Sashimi/MyActionHandler.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Sashimi.Server.Contracts.ActionHandlers;
+using Sashimi.Server.Contracts.Calamari;
+
+namespace Sashimi.NamingIsHard
+{
+    class MyActionHandler : IActionHandler
```

Comment:
Add a validator as well, since it's not discoverable atm

Reply:
Fixed